cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
> That's how I set it in this experiment.

That fallback won't work if the .env file isn't sourced though.

> It seems counter to what environment variables are—essentially constants—to reference them but when they're not set have (potentially different) fallback values set throughout the codebase.

The Django settings file isn't exactly throughout the codebase, in this case (I can't speak for other env vars). If I were coming to the project, that would be the first place I'd be looking for default values, not a hidden file.